pkg/core/blockchain.go

```diff
@@ -524,34 +499,59 @@ func (bc *Blockchain) GetBlock(hash util.Uint256) (*Block, error) {
 	return block, nil
 }
 
+// getBlockFromStore returns Block by the given hash if it exists in the store.
+func getBlockFromStore(s storage.Store, hash util.Uint256) (*Block, error) {
```

Comment:
nit:
noted that in blochchain.go we have now functions which could be put in separate layer which takes care about Storage and getting data from it.
Like in this case we just getting block from store by hash. There's nothing what bc should be care of. Even some parts of the code look pretty same.

Reply:
Yep, that's exactly right and that's the next step planned (and it relates to #334). I've just decided that +463/-377 is enough of git diff to push for now.